ft-sdk/src/in_.rs

```diff
@@ -62,16 +60,13 @@ impl In<'_> {
     ///    form_errors: std::collections::HashMap::new(),
     /// };
     ///
-    /// // the name, domain and path match the cookie created in the previous example
-    /// let mut cookie = Cookie::build("name", "value-does-not-matter")
-    ///     .domain("www.rust-lang.org")
-    ///     .path("/")
-    ///     .finish();
-    ///
+    /// // the name, and path match the cookie created in the previous example
+    /// let mut cookie = Cookie::new("name", "value-does-not-matter");
+    /// cookie.set_max_age(0);
     ///
     /// let res = in_.add_cookie(cookie);
     /// ```
-    pub fn add_cookie<C: Into<cookie::Cookie<'static>>>(&self, cookie: C) {
-        self.set_cookies.borrow_mut().push(cookie.into());
+    pub fn add_cookie(&self, cookie: Cookie) {
```

Comment:
This is not what I had in mind tho. I was thinking `.add_cookie_encoded(String)`, and `.add_cookie(name, value, max_age)`. And we do not export `Cookie` struct at all. End users with user the `.add_cookie()` helper, or add a dependency on `cookie` crate you used earlier, to construct the string version that they pass to `add_cookie_encoded()`.

Reply:
We can also add a `.add_forever_cookie(name, value)` which sets the cookie with far future expiry.